guide/git_flow/git_basics.md

```diff
@@ -3,31 +3,30 @@ Git is a powerful tool that permits parallel and asynchronous work while maximiz
 
 ## The three stages of Git 
 
-:::{figure-md} three-stages
+:::{figure-md} three_stages
```

Comment:
Can you see this image in the docs?  It doesn't render in mine. Maybe changing the name to `three-stages` works better. I'm not yet sure why, but it looks like the docs parser has trouble dealing with underscores.

Reply:
I have introduced multiple changes. Now the page looks better. But it was impossible to render the image. I tried several possible solutions, but nothing worked. Maybe is the file corrupted?